conf/application.conf

```diff
@@ -13,4 +13,4 @@ db.default.username="pathfinderwebserver"
 db.default.password=""
 db.default.logStatements=true
 
-pathfinder.server.socket = "wss://api.thepathfinder.xyz/socket"
+pathfinder.server.socket = "ws://api.thepathfinder.xyz/socket"
```

Comment:
David is correct. But I think I'm going to switch back to Jetty 8.1 soon. It doesn't allow headers, but WSS works and I will use query string AppId.